test/unit/org/apache/cassandra/tools/nodetool/StatusTest.java

```diff
@@ -117,4 +117,51 @@ private void validateStatusOutput(String hostForm, String... args)
         assertThat(hostStatus).endsWith(SimpleSnitch.RACK_NAME);
         assertThat(hostStatus).doesNotContain("?");
     }
-}
+
+    @Test
+    public void testSortByIp() {
```

Comment:
I agree, further testing with multiple hosts would be beneficial to ensure the sorting mechanism is working as expected. Since I haven’t written many test cases yet, I might need some assistance with this. I'll proceed with testing and would appreciate any help to ensure comprehensive coverage.